lib/lint.dart

```diff
@@ -1,14 +0,0 @@
-/// The lint package doesn't ship any dart source code.
-///
-/// To enable `lint`,
-/// 1. Add it to your dev_dependencies
-/// ```yaml
-/// dev_dependencies:
-///   lint: ^1.0.0
-/// ```
-///
-/// 2. Include the rules into your `analysis_options.yaml`
-/// ```yaml
-/// include: package:lint/analysis_options.yaml
-/// ```
-library lint;
```

Comment:
`dart doc` was crashing in the past, causing https://pub.dev/documentation/lint/latest/ not to be generated at all.

It's now replaced with a warning, seems fine

```
Documenting lint...                                                                                                                                                                                                                        warning: package:lint has no documentable libraries
    from package-lint: file:///Users/pascalwelsch/Projects/passsy/dart-lint
            Dartdoc could not find any public libraries to document in
            package:lint, but documentation was requested.  This might be
            expected for an asset only package, in which case, disable this
            warning in your dartdoc_options.yaml file.
Initialized dartdoc with 0 libraries
Found 1 warning and 0 errors.
Documented 0 public libraries in 6.1 seconds
dartdoc could not find any libraries to document
Success! Docs generated into /Users/pascalwelsch/Projects/passsy/dart-lint/doc/api
```

Reply:
TODO: Add `dartdocs_options.yaml` and ignore the `no-documentable-libraries` warning

Reply:
Ah, get it. Thanks for linking to the tickets 👍 

I added a `dartdoc_options.yaml`. Could you review again?

Running `dart doc .` shows this output:
```sh
```